fiftyone/operators/builtin.py

```diff
@@ -1124,44 +1124,59 @@ def config(self):
     def resolve_input(self, ctx):
         inputs = types.Object()
 
-        indexes = ctx.dataset.list_indexes()
+        _drop_index_inputs(ctx, inputs)
 
-        default_indexes = set(ctx.dataset._get_default_indexes())
-        if ctx.dataset._has_frame_fields():
-            default_indexes.update(
-                ctx.dataset._FRAMES_PREFIX + path
-                for path in ctx.dataset._get_default_indexes(frames=True)
-            )
+        return types.Property(inputs, view=types.View(label="Drop index"))
 
-        indexes = [i for i in indexes if i not in default_indexes]
+    def execute(self, ctx):
+        index_names = _to_string_list(ctx.params["index_names"])
 
-        if indexes:
-            index_selector = types.AutocompleteView()
-            for key in indexes:
-                index_selector.add_choice(key, label=key)
+        for index_name in index_names:
+            ctx.dataset.drop_index(index_name)
 
-            inputs.enum(
-                "index_name",
-                index_selector.values(),
-                required=True,
-                label="Index name",
-                description="The index to drop",
-                view=index_selector,
-            )
-        else:
-            prop = inputs.str(
-                "index_name",
-                label="This dataset has no non-default indexes",
-                view=types.Warning(),
-            )
-            prop.invalid = True
 
-        return types.Property(inputs, view=types.View(label="Drop index"))
+def _drop_index_inputs(ctx, inputs):
+    indexes = ctx.dataset.list_indexes()
 
-    def execute(self, ctx):
-        index_name = ctx.params["index_name"]
+    if not indexes:
+        prop = inputs.str(
+            "index_name",
+            label="This dataset has no non-default indexes",
+            view=types.Warning(),
+        )
+        prop.invalid = True
+        return
+
+    default_indexes = set(ctx.dataset._get_default_indexes())
+    if ctx.dataset._has_frame_fields():
+        default_indexes.update(
+            ctx.dataset._FRAMES_PREFIX + path
+            for path in ctx.dataset._get_default_indexes(frames=True)
+        )
+
+    indexes = [i for i in indexes if i not in default_indexes]
 
-        ctx.dataset.drop_index(index_name)
+    index_names = _to_string_list(ctx.params.get("index_names", []))
+
+    index_selector = types.AutocompleteView(multiple=True)
```

Comment:
Ideally can make this `DropdownView(multiple=True)` to avoid manually enforcing valid values below